shared/continuations/lib/LowerAwait.cpp

```diff
@@ -171,7 +171,7 @@ static void processContinuations(
   //    co.flag = llvm.coro.suspend.retcon
   //       unreachable
   auto &Context = M.getContext();
-  auto *I8Ptr = Type::getInt8PtrTy(Context);
+  auto *I8Ptr = PointerType::getUnqual(Context);
```

Comment:
This code should still work in typed pointer mode, so I think we should instead manually construct a pointer to an `i8` by `Type::getInt8Ty(Context)->getPointerTo()`.

Reply:
Done